src/patternfly/components/Page/page.scss

```diff
@@ -453,6 +466,35 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
       width: 100%;
     }
   }
+
+  &.pf-m-nav {
+    --pf-c-page__sidebar-body--PaddingTop: var(--pf-c-page__sidebar-body--m-nav--PaddingTop);
+    --pf-c-page__sidebar-body--PaddingBottom: var(--pf-c-page__sidebar-body--m-nav--PaddingBottom);
+  }
+
+  &.pf-m-inset {
+    padding-right: var(--pf-c-page__sidebar-body--m-inset--PaddingRight);
+    padding-left: var(--pf-c-page__sidebar-body--m-inset--PaddingLeft);
```

Comment:
should we add padding vars to `__body` so we can redeclare those - like top/bottom padding?